src/test/test-execute.c

```diff
@@ -408,6 +408,47 @@ static void test_exec_inaccessiblepaths(Manager *m) {
         test(m, "exec-inaccessiblepaths-mount-propagation.service", can_unshare ? 0 : EXIT_FAILURE, CLD_EXITED);
 }
 
+static void test_exec_mount_apivfs(Manager *m) {
+        _cleanup_strv_free_ char **env = NULL;
+        const char *p;
+        int r;
+
+        FOREACH_STRING(p, "/usr/bin/touch", "/lib64/ld-linux-x86-64.so.2") {
+                r = find_executable(p, NULL);
+                if (r < 0) {
+                        log_notice_errno(r, "Skipping %s, could not find %s binary: %m", __func__, p);
+                        return;
+                }
+        }
+        if (find_executable("/lib64/libc.so.6", NULL) < 0) {
+                r = find_executable("/lib/x86_64-linux-gnu/libc.so.6", NULL);
```

Comment:
what's the purpose of this check? the test will be skipped even if it's successful

Reply:
ouch!